libs/net/template/ReportExtensions.cs

```diff
@@ -372,5 +372,25 @@ public static string GetColorFromName(string targetValue, string[] knownValues,
         }
         else return string.Empty;
     }
+    
+    /// <summary>
+    /// returns a the sum of all child Aggregations
+    /// </summary>
+    /// <param name="aggregations"></param>
+    /// <returns></returns>
+    public static double GetSumOfAggregations(Dictionary<string, AggregationRootModel> aggregations) {
```

Comment:
At some point we're going to have to come up with a way to include all the extension methods in a help file for template builders.